casper/order/order_estimator.py

```diff
@@ -0,0 +1,10 @@
+def get_estimate_from_latest_messages(latest_bets):
+    sample_list = list(latest_bets.values())[0].estimate
+    elem_weights = {elem: 0 for elem in sample_list}
+    for validator in latest_bets:
+        bet = latest_bets[validator]
+        estimate = bet.estimate
+        for i, elem in enumerate(estimate):
+            elem_weights[elem] += validator.weight * (len(estimate) - i)
```

Comment:
So, to show no ties with validator weights, we have to show there are no ties between two subsets of validator weights where weights can be multiplied by any number up to the length of the list - 1. 

I guess this is only a stronger assumption because if we add a random amt to each validator weight, there's a higher chance a tie might exist - maybe :).

Reply:
I don't think there's a higher chance of tie. Each element has only one contribution from each validator. That contribution is V.weight*M. This contribution looks like a new jittered weight itself - Y_i. So for each element we have a set of Y_i's that don't look like the set of Y_j's from any other element ==> sum(Y_i's) doesn't look like sum(Y_j's).

That's a terrible proof. I'll write it better and put it in the estimator comments or wiki.